app/libs/classes/viva_application.py

```diff
@@ -119,15 +141,57 @@ def get_application_status(self):
     def _get_application(self):
         initial_application = {
             'OTHER': '',
-            'RAWDATA': "",
-            'RAWDATATYPE': "PDF",
+            'RAWDATA': self._raw_data,
+            'RAWDATATYPE': self._raw_data_type,
             'HOUSEHOLDINFO': ''
         }
 
         application = self._answers_to_zeep_dict()
 
         return {**initial_application, **application}
 
+    def _save_completion_attachments(self):
+        try:
+            for attachment in self._attachments:
+                self._viva_attachments.save(attachment=attachment)
+            return True
```

Comment:
does not need to return true, if errors occurs in the loop an exception will be raised that we can handle upstream.